app/scripts/app.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+  function config($locationProvider, $stateProvider) {
+         $locationProvider
+             .html5Mode({
+                 enabled: true,
+                 requireBase: false
+              });
+
+         $stateProvider
+             .state('home', {
+                 url: '/',
+                 controller: 'HomeCtrl as home',
+                 templateUrl: '/templates/home.html'
+             });
+     }
+
+     angular
+         .module('capstone', ['ui-router'])
```

Comment:
This should be `ui.router`.

Reply:
Ahhh I knew it was going to be something minor like that!